cypress/integration/challenge-tests/01-responsive-web-design/breadcrumbs.js

```diff
@@ -0,0 +1,18 @@
+/* global cy */
+
+let structure = {
+  challenge: []
+};
+
+describe('Test breadcrumbs', () => {
+  it('should have the correct link', () => {
+    cy.task('getAllChallengePaths', {
+      superBlock: '01-responsive-web-design',
+      lang: 'english'
+    }).then(res => {
+      let { challenge } = structure;
+      challenge = res;
+      console.log(challenge);
+    });
```

Comment:
```suggestion
      lang: 'english'
    }).then(curriculum => {
      console.log(curriculum);
    });
```
Rather than using `structure` as a cache, I think it makes sense to do that in the plugin.  That way you can call the task wherever you like, without worrying that it'll slow the process down.

Something like

```js
if(!curriculum) {
  curriculum = getChallengesForLang(lang)
}
return curriculum;
```
should do the trick.